tests/e2e/tests-library/WorkspaceHandlingTests.ts

```diff
@@ -66,6 +66,19 @@ export class WorkspaceHandlingTests {
         });
     }
 
+    public createAndOpenWorkspaceFromFactory(factoryUrl: string): void {
+        test(`Create and open new workspace from factory:${factoryUrl}`, async () => {
+            await this.dashboard.waitPage();
+            Logger.debug(`Fetching user kubernetes namespace, storing auth token by getting workspaces API URL.`);
+            await this.apiUrlResolver.getWorkspacesApiUrl();
+            await this.dashboard.clickCreateWorkspaceButton();
+            await this.createWorkspace.waitPage();
+            WorkspaceHandlingTests.parentGUID = await this.browserTabsUtil.getCurrentWindowHandle();
+            await this.createWorkspace.startWorkspaceUsingFactory(factoryUrl);
```

Comment:
There are another steps in test scenario after entering "Create Workspace" page: https://github.com/eclipse/che/issues/22031 : 
```
2. In the `Import from Git` section input https://github.com/che-incubator/quarkus-api-example
3. Click `Create & Open Button`
```

Reply:
Sorry, I misunderstood the code.

The name of method is misleading, IMHO: `startWorkspaceUsingFactory(factoryUrl);` https://github.com/eclipse/che/blob/main/tests/e2e/pageobjects/dashboard/CreateWorkspace.ts#L55

It's literally using section `Import from Git`, which is not obvious.